scripts/build/build_node/Platform/Windows/install_vsbuildtools.ps1

```diff
@@ -5,9 +5,9 @@ For complete copyright and license terms please see the LICENSE at the root of t
 SPDX-License-Identifier: Apache-2.0 OR MIT
 #>
 
-choco install visualstudio2022buildtools --version=17.2.6 --package-parameters "--config .\vs2022bt.vsconfig" -y
-choco install visualstudio2019buildtools --version=16.8.2 --package-parameters "--config .\vs2019bt.vsconfig" -y
-choco install visualstudio2017buildtools --version=15.9.29 --package-parameters "--config .\vs2017bt.vsconfig" -y
+choco install visualstudio2022buildtools --version=17.2.10 --package-parameters "--config .\vs2022bt.vsconfig" -y
+choco install visualstudio2019buildtools --version=16.11.21 --package-parameters "--config .\vs2019bt.vsconfig" -y
+choco install visualstudio2017buildtools --version=15.9.51 --package-parameters "--config .\vs2017bt.vsconfig" -y
```

Comment:
We don't need VS2017 anymore.
It is unsupported.

Reply:
Sure thing. Removing it

Reply:
Correct, the engine will not even build with VS2017 anymore